app/components/back-office-app-services/variables.tf

```diff
@@ -183,3 +183,8 @@ variable "tags" {
   description = "The tags applied to all resources"
   type        = map(string)
 }
+
+variable "feature_service_bus_enabled" {
+  description = "Whether or not Service Bus events are enabled"
+  type        = string
+}
```

Comment:
would this not be a boolean?

Reply:
It will be interpreted as one, but this gets spat out to app_settings in the node-app-service module which is a map(string)